tools/accuracy_checker/accuracy_checker/metrics/coco_orig_metrics.py

```diff
@@ -107,7 +108,56 @@ def _prepare_coco_structures(self, annotations):
 
         return coco
 
-    def _convert_data_to_coco_format(self, predictions):
+    @staticmethod
+    def generate_map_pred_label_id_to_coco_cat_id(has_background, use_full_label_map):
+        shift = 0 if has_background else 1
+        max_cat = 90 if use_full_label_map else 80
+        max_key = max_cat - shift
+        res_map = {i: i + shift for i in range(0, max_key+1)}
+        assert max(res_map.values()) == max_cat
+        return res_map
+
+    def _prepare_original_coco_structures(self):
+        annotation_conversion_parameters = self.dataset.config.get('annotation_conversion')
+        annotation_file = annotation_conversion_parameters.get('annotation_file')
+        has_background = annotation_conversion_parameters.get('has_background', False)
+        use_full_label_map = annotation_conversion_parameters.get('use_full_label_map', False)
+
+        meta = self.dataset.metadata
+
+        if COCO is None:
```

Comment:
If I right remember we move this check to configure, is not it?
If yes, possibly we do not need to check it twice

Reply:
there are checks for self.dataset.metadata and self.dataset.metadata.get('label_map') in configure. I can move check for COCO in configure too